src/main/java/com/nike/vault/client/VaultClient.java

```diff
@@ -313,6 +333,8 @@ protected Response execute(final HttpUrl url, final String method, final Object
      * @param response Response to parses the error details from
      */
     protected void parseAndThrowErrorResponse(final Response response) {
+        logger.debug(LOG_HTTP_RESPONSE_BODY_TMPL, gson.toJson(response.body()));
```

Comment:
@sdford I think in this method under the catch block you might add this same statement as a WARN or ERROR level log.